packages/plugin-react-native-global-error-handler/error-handler.js

```diff
@@ -0,0 +1,22 @@
+/*
+* Automatically notifies Bugsnag when React Native's global error handler is called
+*/
+
+const createReportFromErr = require('@bugsnag/core/lib/report-from-error')
+
+module.exports = {
+  init: (client, ErrorUtils = global.ErrorUtils) => {
+    if (!client.config.autoNotify) return
```

Comment:
Would `ErrorUtils` be present in end users' test environments? Is it something that would require a null check?  I ask because I had this check previously in `bugsnag-react-native` and don't remember why. 😅

Reply:
> Would `ErrorUtils` be present in end users' test environments?
I'm not sure.

I don't see a reason not to add a check and a log warning if it's not defined so I'll do that.